includes/definitions/hpe-mapdu.yaml

```diff
@@ -0,0 +1,21 @@
+os: hpe-mapdu
+text: 'HPE Managed Power Distribution Unit'
+type: power
+icon: hpe
+group: hpe
+mib_dir: hp
+over:
+    - {graph: device_power, text: Power}  
+    - {graph: device_current, text: Current}
+snmp_max_oid: '8'
+discovery:
+    -
+        sysObjectID:
+            - .1.3.6.1.4.1.232.165.7
+            - .1.3.6.1.4.1.232.165.11
+    -        
+        sysDescr:
```

Comment:
These sysDescr seem a little generic.  Do you have devices that do not supply sysObjectID? (if no, just remove sysDescr detection)

Reply:
Understand, I will make them more specific, without sysDescr I had trouble to pass the tests as devices were reported only as generic instead hpe-mapdu during tests. In real installation correct detection took quite a while.

Main problem was on PDU3  (.1.3.6.1.4.1.232.165.11) type of devices

Reply:
Change made, my .1.3.6.1.4.1.232.165.11 devices are not presenting sysObjectID correctly and thus is not recognized by the discovery code and device is marked as generic.
Devices are producing String value instead of OID value type.
`.1.3.6.1.2.1.1.2.0 = Wrong Type (should be OBJECT IDENTIFIER): STRING: ".1.3.6.1.4.1.232.165.11"`

Reply:
Odd, what does the sysObject output look like with the `-Oqv` options?

Below sysDescr is way to specific now :smile:  
Maybe something like `Outlet Metered, Outlet Switched`

Reply:
:smile: will change it.
Bellow test for pdu2 and pdu3 with the options:
**PDU2**
`snmpget -Oqv -v1 -c public -m MIB:all -M +/opt/librenms/mibs/hp/ 172.17.30.80 1.3.6.1.2.1.1.2.0`
`CPQPOWER-MIB::pdu2`

**PDU3**
`snmpget -Oqv -v2c -c public -m MIB:all -M +/opt/librenms/mibs/hp/ 10.40.254.90 1.3.6.1.2.1.1.2.0`
`Wrong Type (should be OBJECT IDENTIFIER): ".1.3.6.1.4.1.232.165.11" `